src/api/handlers/member.ts

```diff
@@ -276,3 +276,19 @@ export async function editBotProfile(username?: string, botAvatarURL?: string) {
     },
   );
 }
+
+/** Edit the nickname of the bot in this guild */
+export async function editBotNickname(
+  guildID: string,
+  nickname?: string | null,
+) {
+  const hasPerm = await botHasPermission(guildID, ["CHANGE_NICKNAME"]);
+  if (!hasPerm) throw new Error(Errors.MISSING_CHANGE_NICKNAME);
+
+  const newNickname = (await RequestManager.patch(
+    endpoints.USER_NICK(guildID),
+    { nick: nickname },
+  )).nick as string;
+
+  return newNickname;
```

Comment:
```suggestion
  const resp = await RequestManager.patch(
    endpoints.USER_NICK(guildID),
    { nick: nickname },
  );

  return resp.nick as string;
```

Reply:
For consistency var should be named `response`

Reply:
I don't mind, go ahead.